app/concepts/matestack/ui/core/collection/order/order.rb

```diff
@@ -0,0 +1,17 @@
+module Matestack::Ui::Core::Collection::Order
+  class Order < Matestack::Ui::Core::Component::Dynamic
+
+    def setup
+      @component_config = @component_config.except(:data, :paginated_data)
+    end
+
+    def response
+      components {
+        div do
```

Comment:
Can this `div` be savely removed?

Reply:
nope. otherwise vue.js will complain as this is a dynamic component template